coast/data/tidegauge.py

```diff
@@ -104,13 +127,21 @@ def read_gesla_v3(self, fn_gesla, date_start=None, date_end=None):
         fn_gesla (str) : path to gesla tide gauge file, list of files or a glob
         date_start (datetime) : start date for returning data
         date_end (datetime) : end date for returning data
+        format (str) : accepts "v3" or "v5"
 
         Returns
         -------
         Creates xarray.dataset within tidegauge object containing loaded data.
         If multiple files are provided then instead returns a list of NEW
         tidegauge objects.
         """
+        if format != "v3" and format != "v5":
+            debug(f"Not expecting GESLA text format {format}")
+            print(f"Not expecting GESLA text format {format}")
+
+        if format == "v5":
+            print(f"Not written code for format {format}")
```

Comment:
You can raise a NotImplementedError to tell people that a piece of code has yet to be written:
e.g.:
```python
if format == "v5":
    raise NotImplementedError(f"Not written code for format {format}")
```